README.md

```diff
@@ -114,9 +114,9 @@ job.batch/volcano-admission-init   1/1           48s        96s
 
 You can reach the maintainers of this project at:
 
-Slack: [#volcano-sh](volcano-slack). (Signup [here](slack-invitation)).
+Slack Channel : https://volcano-sh.slack.com
```

Comment:
something like that:

Slack Channel : [https://volcano-sh.slack.com](https://volcano-sh.slack.com) , ([Signup](http://shorturl.at/dFIKQ))

Mailing List: [https://groups.google.com/forum/#!forum/volcano-sh](https://groups.google.com/forum/#!forum/volcano-sh)

Reply:
btw, it seem can not access shorturl.at in china.

Reply:
so shall i keep as below because short url is not working in china.

Slack Channel : https://volcano-sh.slack.com.[Slack Signup](https://join.slack.com/t/volcano-sh/shared_invite/enQtNTU5NTU3NDU0MTc4LTgzZTQ2MzViNTFmNDg1ZGUyMzcwNjgxZGQ1ZDdhOGE3Mzg1Y2NkZjk1MDJlZTZhZWU5MDg2MWJhMzI3Mjg3ZTk) 
 

Mailing List  : https://groups.google.com/forum/#!forum/volcano-sh

Reply:
ok